sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/persistence/DatabasePersistenceAndroidTest.java

```diff
@@ -322,8 +323,6 @@ public void putLargeLogNotSupportedOnCommonSchema() throws JSONException {
                 assertEquals(0, persistence.countLogs("test-p1"));
             }
         } finally {
-
-            /* Close. */
```

Comment:
nit - what is the reason of removing this comment everywhere? Looks like to me it's fine to keep it.